### DISCUSSION TITLE
RARBG repository

### DISCUSSION THREAD

**Discussion by Damaux** [LivingWithHippos/unchained-android#318] [Q&A]
Hi there, so now that RARBG is down they have people having scraped data from it and putting on github as a repository or on database like sqlite software. I found a repository from github that has all the magnets for everything the site had. I was wondering how can I add it to the plugins section on the app. I tried adding the link but tells me network error. Any ideas how to do this please. Thanks!

**Comment by Damaux** [comment]
Source:
https://github.com/2004content/rarbg

**Reply by LivingWithHippos** [reply]
To be honest the repo is going down, no way github is going to leave it be.

Also:
- some files are compressed so we cannot use them
- the files with the magnets do not have any info about the content so I wouldn't know how to search for something (there may be the title on the torrent)
- the files are big so everytime you search the whole file will be downloaded

What will happen
- the rarbg plugin will be removed
- when a searchable version of rarbg establish itself we can create a new plugin


In the meanwhile if you want to try something you should open the file with the magnets in github and press the "raw" button. The link you are transported to is the one you should use. I suggest you to wait anyway.

**Comment by Damaux** [comment]
Ok thanks for letting me know!
